docs/usage/modules/index.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Do we need this frontmatter?

```markdown
---
title: Modules
---
```

The markdown file has a H1 heading, which I guess/hope would be enough for Material for MkDocs to understand what title the page should have?

Read the [Material for MkDocs, reference, setting the page title](https://squidfunk.github.io/mkdocs-material/reference/#setting-the-page-title) to learn more.

Reply:
maybe you're right. will check it tomorrow

Reply:
if i remove it, then the pages plugin uses `Introduction` from `.pages` as page title, which isn't wanted here